ex-3542-sample-attestation-doc-format (#153)
DIMO-Network/dis: README.md

### Attestation Data Format

```json
{
  "id": "unique-event-identifier",
  "source": "0x07B584f6a7125491C991ca2a45ab9e641B1CeE1b",
  "producer": "did:ethr:80002:0x07B584f6a7125491C991ca2a45ab9e641B1CeE1b",
  "specversion": "1.0",
  "subject": "did:nft:80002:0x45fbCD3ef7361d156e8b16F5538AE36DEdf61Da8_847",
  "time": "2025-04-14T15:02:53.83882-04:00",
  "type": "dimo.attestation",
  "signature": "0x828b1c357fbab66524aca14f3d7389beedb4d6fb07a1c5df74b28d0081a2ba8b0ee0f886bb8294ed47334ce6130989db0d4f81c91b0e111bcdbb2c7cc8e84f2301",
  "data": {
    "subject": "did:nft:80002:0x45fbCD3ef7361d156e8b16F5538AE36DEdf61Da8_847",
    "attestorAddress": "0x07B584f6a7125491C991ca2a45ab9e641B1CeE1b",
    "insured": true,
    "provider": "State Farm",
    "policyNumber": "SF-12345678",
    "policyTerms": "terms-of-insurance",
    "coverageType": "Liability + Collision",
    "coverageStartDate": 1744751357,
    "expirationDate": 1807822654
  }
}
```

### Attestation Cloud Event Header Descriptions

- **source**: Required field. The connection license address. Note that this field will be overwritten with the connection license address pulled from the CN of the certificate used for authentication.

> - **subject**: Required field. The NFT DID which denotes which vehicle token ID is being used. Must follow the format `did:<chain>:<chainId>:<contractAddress>`.

Reviewer: ```suggestion
- **subject**: Required field. The NFT DID which denotes which vehicle token ID the attestation is about. Must follow the format `did:<chain>:<chainId>:<contractAddress>`.
```
